tests/FSharp.AWS.DynamoDB.Tests/SimpleTableOperationTests.fs

```diff
@@ -256,30 +257,30 @@ type ``TransactWriteItems tests``(table1: TableFixture, table2: TableFixture) =
     let ``All paths`` shouldFail = async {
         let item, item2, item3, item4, item5, item6, item7 = mkItem (), mkItem (), mkItem (), mkItem (), mkItem (), mkItem (), mkItem ()
         let! key = table1.PutItemAsync item
-        let transactionBuilder = TransactionBuilder()
+        let Transaction = Transaction()
 
         let requests =
-            [ transactionBuilder.Update(table1, key, compileUpdateTable1 <@ fun t -> { t with Value = 42 } @>, existsConditionTable1)
-              transactionBuilder.Put(table1, item2)
-              transactionBuilder.Put(table1, item3, doesntExistConditionTable1)
-              transactionBuilder.Delete(table1, table1.Template.ExtractKey item4, Some doesntExistConditionTable1)
-              transactionBuilder.Delete(table1, table1.Template.ExtractKey item5, None)
-              transactionBuilder.Check(
+            [ Transaction.Update(table1, key, compileUpdateTable1 <@ fun t -> { t with Value = 42 } @>, existsConditionTable1)
+              Transaction.Put(table1, item2)
+              Transaction.Put(table1, item3, doesntExistConditionTable1)
+              Transaction.Delete(table1, table1.Template.ExtractKey item4, Some doesntExistConditionTable1)
+              Transaction.Delete(table1, table1.Template.ExtractKey item5, None)
+              Transaction.Check(
```

Comment:
The value naming threw me here a little, should this be `transaction` instead of `Transaction`?